ngi_reports/utils/entities.py

```diff
@@ -473,8 +475,10 @@ def get_library_method(self, project_name, application, library_construction_met
                         lib_list.append('* {}: {}'.format(name, value))
                 return ('\n'.join(lib_list))
             else:
-                log.error('Library method is not mentioned in expected format for project {}'.format(project_name))
-                return None
+                if library_prep_option:
```

Comment:
We will have to update the pattern here `lib_meth_pat = r'^(.*?),(.*?),(.*?),(.*?)[\[,](.*)$' #Input, Type, Option, Category -/, doucment number` now that the values are all updated. This can be a temporary solution until all the projects with the old values are closed.